<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>help_guide</class>
 <widget class="QDialog" name="help_guide">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>155</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>145</width>
     <height>215</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>145</width>
      <height>130</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>5</y>
       <width>145</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>我的QQ中心</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>35</y>
       <width>145</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>使用帮助</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>65</y>
       <width>145</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>腾讯客服</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>95</y>
       <width>145</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>反馈问题</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>130</y>
      <width>147</width>
      <height>45</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>5</y>
       <width>145</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>QQ新特性</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>175</y>
      <width>145</width>
      <height>37</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>5</y>
       <width>145</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>QQ官网</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
